konan/konan.properties

```diff
@@ -244,6 +262,9 @@ linkerDynamicFlags.watchos_arm32 = -dylib
 osVersionMinFlagLd.watchos_arm32 = -watchos_version_min
 osVersionMinFlagClang.watchos_arm32 = -mwatchos-version-min
 osVersionMin.watchos_arm32 = 5.0
+runtimeDefinitions.watchos_arm32 = KONAN_OBJC_INTEROP=1 KONAN_WATCHOS KONAN_ARM32=1 \
```

Comment:
Minor: May be add a comment about `KONAN_NO_64BIT_ATOMIC` here and for `watchos_arm64` like it was [done](https://github.com/JetBrains/kotlin-native/pull/4439/files#diff-966768f5d907237e2d7b0532bb7287949e023968f4c967b2a1461d6dba5e9c6dL273) in `ClangArgs.kt`?